terraform/prod/main.tf

```
@@ -455,12 +455,13 @@ module "stagecoach_vpn" {
 module "internal_avl_ingestion" {
   source = "../modules/data-pipelines/internal-avl-ingestion"
 
-  environment                 = local.env
-  vpc_id                      = module.integrated_data_vpc.vpc_id
-  lb_subnet_ids               = module.integrated_data_vpc.private_subnet_ids
-  external_ip_range           = local.secrets["stagecoach_destination_cidr_block"]
-  data_endpoint_function_name = module.integrated_data_avl_data_producer_api.avl_data_endpoint_function_name
-  nlb_ip_address              = local.secrets["internal_avl_ingestion_nlb_ip"]
+  environment                               = local.env
+  vpc_id                                    = module.integrated_data_vpc.vpc_id
+  lb_subnet_ids                             = module.integrated_data_vpc.private_subnet_ids
+  external_ip_range                         = local.secrets["stagecoach_destination_cidr_block"]
+  data_endpoint_function_name               = module.integrated_data_avl_data_producer_api.avl_data_endpoint_function_name
```

```suggestion
  avl_data_endpoint_function_name               = module.integrated_data_avl_data_producer_api.avl_data_endpoint_function_name
```

Could be nice to rename this variable to distinguish between avl and cancellations data endpoints